app/scss/addQuestion.scss

```diff
@@ -0,0 +1,91 @@
+@import 'imports/variables';
+@import url('https://fonts.googleapis.com/css?family=Asap&display=swap');
+
+* {
+  box-sizing: border-box;
+
+  body {
+    font-family: 'Asap', sans-serif;
+
+    .container {
+
+      .form {
+        margin-top: 100px;
+      }
+
+      .logo {
+        margin: 2%;
+        width: 300px;
+        float: right;
+      }
+
+      textarea {
+        resize: vertical;
+      }
+
+      .answers {
+        margin-top: 5%;
+        width: 97%;
+      }
+
+      .question {
+        margin: 5%;
+        width: 90%;
+      }
+
+      .option {
+        margin: 1% 5%;
+        width: 90%;
+      }
+
+      .answer-check {
+        margin-top: -40px;
+        margin-right: -30px;
+        transform: scale(2);
```

Comment:
height/width can be set instead?

Reply:
Doesn't allow you to set the height or width on check inputs.